samples/meetings-context-app/csharp/MeetingContextApp/ClientApp/package.json

```diff
@@ -9,8 +9,8 @@
     "@testing-library/react": "^11.2.7",
     "@testing-library/user-event": "^12.8.3",
     "msteams-ui-components-react": "^0.8.4",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
+    "react": "^16.8.0",
```

Comment:
Why we have degraded this?
Please share the RCA for it.

Reply:
msteams-ui-components-react package is not supported with latest react package, so degraded it.